app/middleware/locationValidator.js

```diff
@@ -9,37 +12,56 @@ function setLocationLabel(res, location) {
   }
 }
 
+function renderFindHelpPage(req, res, location, message, errorMessage) {
+  log.info({ locationValidationResponse: { location } }, message);
+  res.locals.errorMessage = errorMessage;
+  setLocationLabel(res, location);
+  renderer.findHelp(req, res);
+}
+
 function validateEnglishLocation(req, res, next) {
   const location = res.locals.location;
-
   const validationResult = locationValidator(location);
-
   res.locals.location = validationResult.alteredLocation;
-
   if (validationResult.errorMessage) {
-    log.info({ locationValidationResponse: { location } }, 'Non-English postcode');
-    res.locals.errorMessage = validationResult.errorMessage;
-    setLocationLabel(res, location);
-    renderer.findHelp(req, res);
+    renderFindHelpPage(req, res, location, 'Non-English postcode', validationResult.errorMessage);
   } else {
     next();
   }
 }
+function sanitiseString(string) {
+  return string.replace(/[^a-z]/gmi, ' ').replace(/\s\s+/g, ' ').trim();
```

Comment:
This looks very similar to https://github.com/nhsuk/connecting-to-services/pull/369/files#diff-e675aab0ca0472369ec07b2b8552e5a9R13. Perhaps they can be extracted into a utility method?